<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>474</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>512</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="Layout">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="QFrame" name="frame">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>60</height>
         </size>
        </property>
        <property name="mouseTracking">
         <bool>false</bool>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="panBtn">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Pen</string>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="colorBtn">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Color</string>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="eBtn">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Eraser</string>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="runBtn">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>RUN</string>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="horizontalFrame">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>360</width>
          <height>16777215</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>25</height>
            </size>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="text">
            <string>penSize : </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="penSizeLabel">
           <property name="minimumSize">
            <size>
             <width>17</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>11</width>
             <height>25</height>
            </size>
           </property>
           <property name="font">
            <font>
             <underline>true</underline>
            </font>
           </property>
           <property name="toolTipDuration">
            <number>-1</number>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="text">
            <string>2</string>
           </property>
           <property name="textFormat">
            <enum>Qt::AutoText</enum>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="penSizeSlider">
           <property name="maximumSize">
            <size>
             <width>250</width>
             <height>25</height>
            </size>
           </property>
           <property name="minimum">
            <number>2</number>
           </property>
           <property name="maximum">
            <number>10</number>
           </property>
           <property name="sliderPosition">
            <number>2</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="pframe">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>30</y>
         <width>503</width>
         <height>102</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="VBOX">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetMinAndMaxSize</enum>
        </property>
        <property name="leftMargin">
         <number>10</number>
        </property>
        <property name="topMargin">
         <number>10</number>
        </property>
        <property name="rightMargin">
         <number>10</number>
        </property>
        <property name="bottomMargin">
         <number>10</number>
        </property>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>512</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="fileMenu">
    <property name="title">
     <string>F&amp;ile</string>
    </property>
    <addaction name="fileOpen"/>
    <addaction name="fileSave"/>
   </widget>
   <widget class="QMenu" name="menupenSize">
    <property name="title">
     <string>pe&amp;nSize</string>
    </property>
    <addaction name="penSize2"/>
    <addaction name="penSize3"/>
    <addaction name="penSize4"/>
    <addaction name="penSize5"/>
    <addaction name="penSize6"/>
    <addaction name="penSize7"/>
   </widget>
   <addaction name="fileMenu"/>
   <addaction name="menupenSize"/>
  </widget>
  <action name="fileOpenBtn">
   <property name="text">
    <string>&amp;open</string>
   </property>
  </action>
  <action name="fileSaveBtn">
   <property name="text">
    <string>&amp;save</string>
   </property>
  </action>
  <action name="action1">
   <property name="text">
    <string>1</string>
   </property>
  </action>
  <action name="penSize2">
   <property name="text">
    <string>&amp;2</string>
   </property>
  </action>
  <action name="penSize3">
   <property name="text">
    <string>&amp;3</string>
   </property>
  </action>
  <action name="penSize4">
   <property name="text">
    <string>&amp;4</string>
   </property>
  </action>
  <action name="penSize5">
   <property name="text">
    <string>&amp;5</string>
   </property>
  </action>
  <action name="penSize6">
   <property name="text">
    <string>&amp;6</string>
   </property>
  </action>
  <action name="penSize7">
   <property name="text">
    <string>&amp;7</string>
   </property>
  </action>
  <action name="fileOpen">
   <property name="text">
    <string>&amp;open (Crtl+O)</string>
   </property>
  </action>
  <action name="fileSave">
   <property name="text">
    <string>&amp;save</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>panBtn</tabstop>
  <tabstop>eBtn</tabstop>
  <tabstop>colorBtn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
